jest.config.mjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+/** @type {import('jest').Config} */
+const config = {
+  testEnvironment: "node",
+  testMatch: ["**/*.test.js"],
+  coverageDirectory: "coverage",
+  coveragePathIgnorePatterns: [
+    "cli\\.js$",
+    ".*config\\.js$",
+    ".*/coverage/.*",
+    ".*/node_modules/.*",
+  ],
+  collectCoverageFrom: ["**/*.js"],
+  coverageProvider: "v8",
+  setupFiles: ["./jest.setup.mjs"],
+  transform: {},
```

Comment:
Maybe remove this property

Reply:
It's needed to avoid jest from transforming my test code to cjs

https://jestjs.io/docs/ecmascript-modules

![image](https://github.com/heliomar-pena/i18n-populator/assets/66505715/e5e72e13-3491-4caa-9dbb-dc642e42df19)

Reply:
Ah got it, super weird eh? Hehe but no prob